src/i18n/i18n.ts

```diff
@@ -0,0 +1,29 @@
+import i18n from 'i18next';
```

Comment:
rename to i18nInit?

Reply:
I also first thought about init. However the default export from this file `i18n`-instance/promise is passed to the provider in BrowserApp. What do you think?

Reply:
You can name them as anything when you import it, default export is flexible
`import i18n from ./i18nInit.ts`